task_definition/container_with_sidecar/task_definition.tf

```diff
@@ -4,7 +4,17 @@ module "task_role" {
   task_name = var.task_name
 }
 
+# The presence of two almost identical blocks of config is working
```

Comment:
This is no longer true: https://www.terraform.io/docs/configuration/expressions.html#dynamic-blocks

Reply:
Ooh, interesting.

Reply:
🚀

Reply:
Hmm. Having read how dynamic blocks work, I’m inclined to leave it as-is.

You can do dynamic blocks, but it’s a bit fiddly – we don’t want “0 to N”, we want “on or off”. We could make it work, but you're shoe-horning a boolean into a loop.

How would you feel about me leaving a comment “we looked at dynamic blocks, weren't right last time we looked, but keep an eye on them in future”?